src/config.ts

```diff
@@ -0,0 +1,55 @@
+import { providers } from "ethers";
```

Comment:
@0xZenus bring this closer to usage. Place under hooks/useMellowLPVaults directory along side your hook.

Reply:
I was thinking to treat it as a global configuration file in the future. Maybe we can process all env variable in here at once. What do you think? There's a lot to off-load (factory address, network names, ids, whitelisted addresses etc.)

Reply:
sounds good but until we do that lets move it closer to usage, since it is easier to delete :) You can always move it away once you start using it across project

Reply:
agree.

Reply:
https://github.com/Voltz-Protocol/voltz-ui/pull/372/commits/688bc5da2bec4d30c58dc1afb2d4401b10fc4e2b